packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card.tsx

```diff
@@ -1,103 +1,76 @@
-import React from 'react';
 import classNames from 'classnames';
-import { Icon, Text, Popover } from '@deriv/components';
-import { Localize, localize } from '@deriv/translations';
-import { getJurisdictionContents } from '../../Constants/jurisdiction-contents';
-import { TJurisdictionCardProps, TJurisdictionCardType } from '../props.types';
-import JurisdictionCardBanner from './jurisdiction-card-banner';
+import React from 'react';
+import { Jurisdiction } from '@deriv/shared';
+import { getJurisdictionContents } from '../../Constants/jurisdiction-contents/jurisdiction-contents';
+import { TJurisdictionCardProps } from '../props.types';
+import JurisdictionCardBack from './jurisdiction-card-back';
+import JurisdictionCardFront from './jurisdiction-card-front';
 
 const JurisdictionCard = ({
+    account_status,
     account_type,
     disabled,
-    context,
+    is_onfido_design = false,
     jurisdiction_selected_shortcode,
-    financial_available_accounts,
     setJurisdictionSelectedShortcode,
-    synthetic_available_accounts,
-    swapfree_available_accounts,
     type_of_card,
 }: TJurisdictionCardProps) => {
     const card_classname = `cfd-jurisdiction-card--${account_type}`;
-    const number_of_synthetic_accounts_to_be_shown = synthetic_available_accounts?.length;
-    const number_of_financial_accounts_to_be_shown = financial_available_accounts?.length;
-    const number_of_swapfree_accounts_to_be_shown = swapfree_available_accounts?.length;
-
     const is_synthetic = account_type === 'synthetic';
     const is_swapfree = account_type === 'all';
-    const non_synthetic_accounts = is_swapfree
-        ? number_of_swapfree_accounts_to_be_shown
-        : number_of_financial_accounts_to_be_shown;
-    const [number_of_cards] = React.useState(
-        is_synthetic ? number_of_synthetic_accounts_to_be_shown : non_synthetic_accounts
-    );
-
-    const card_values = getJurisdictionContents()[type_of_card as TJurisdictionCardType];
-
-    const non_synthetic_card_data = is_swapfree ? card_values.swapfree_contents : card_values.financial_contents;
-    const card_data = is_synthetic ? card_values.synthetic_contents : non_synthetic_card_data;
+    const card_values = getJurisdictionContents()[type_of_card];
+    const non_synthetic_card_data = is_swapfree
+        ? card_values?.swapfree_contents ?? []
+        : card_values?.financial_contents;
+    const card_data = is_synthetic ? card_values?.synthetic_contents : non_synthetic_card_data;
+    const [is_card_flipped, setIsCardFlipped] = React.useState(false);
+    const is_card_selected = jurisdiction_selected_shortcode === type_of_card;
+    let verification_docs = is_synthetic
+        ? card_values?.synthetic_verification_docs
+        : card_values?.financial_verification_docs;
```

Comment:
For swap free- this is giving financial verification docs, which is little confusing. It doesn't break anything now. But can we improve this to avoid confusion?

Reply:
@amina-deriv these changes are from swap-free commits.